File: 프로그래머스/lv1/12977. 소수 만들기/소수 만들기.py
def solution(nums):
    answer = 0
    sums = []
    
    for i in range(len(nums)-2):
        for j in range(i+1, len(nums)-1):
            for k in range(j+1, len(nums)):
                sums.append(nums[i] + nums[j] + nums[k])
    
    for i in sums:
        for j in range(2, int(i ** 0.5) + 1):
            if i % j == 0:
                check = False
                break
        else:
            answer += 1
    
    return answer

File: 프로그래머스/1/12977. 소수 만들기/소수 만들기.js
function isPrime(nums) {
    const prime = new Set()
    for (let i = 1; i <= Math.sqrt(nums); i++) {
        if (nums % i === 0) {
            prime.add(i)
            prime.add(nums / i)
        }
    }
    if (prime.size === 2) return true
    else return false
}

function solution(nums) {
    var answer = 0;
    
    for(let i = 0; i < nums.length - 2; i++) {
        for(let j = i + 1; j < nums.length - 1; j++) {
            for(let k = j + 1; k < nums.length; k++) {
                if (isPrime(nums[i] + nums[j] + nums[k])) answer++
            }
        }
    }

    
    return answer;
}